packages/config/src/config/routes.ts

```diff
@@ -72,7 +73,17 @@ export default function getRoutes() {
     {
       method: 'GET',
       path: '/config',
-      handler: applicationConfigHandler,
+      handler: configHandler,
+      config: {
+        auth: false,
```

Comment:
Must remove auth:false to require the JWT, and complete this part of the ticket:

"Config microservice should check the supplied JWT and only return certificate SVG if the user is a Registration Agent or Registrar, ie has the "register" or "certify" scope"

I do not see this implemented.  Field Agents can get access to the certificate SVG